.travis.yml

```diff
@@ -6,6 +6,7 @@ compiler:
 addons:
   apt:
     packages:
+    - texinfo
```

Comment:
Does the RTEMS build chain really need texinfo now to run?

Reply:
Last I checked, gcc builds its documentation unconditionally.

Reply:
I tried to build the RTEMS on travis. That asked for texinfo. I gave it up because building the newlib took too long for the travis :-(

downloading: sources/mpc-1.0.3.tar.gz - 654.2kB of 654.2kB (100%) 
download: https://ftp.gnu.org/gnu/gmp/gmp-6.1.0.tar.bz2 -> sources/gmp-6.1.0.tar.bz2
downloading: sources/gmp-6.1.0.tar.bz2 - 2.3MB of 2.3MB (100%) 
building: i386-rtems5-gcc-7.4.0-newlib-1d35a003f-x86_64-linux-gnu-1
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated

Reply:
I tried this previously.  With RTEMS 4.9, both the i386 and powerpc toolchains build.  With RTEMS 4.10, the i386 toolchain builds, but the powerpc takes too long and times out.  In my initial attempt with 4.11, both timed out.  I'm guessing that RTEMS 5 toolchain takes even longer.

Reply:
I would suggest building on a local Ubuntu VM.

Reply:
I did that and built RTEMS on a Ubuntu system. The ravis-Build.sh loads it from
curl -L https://dwnld.rz-berlin.mpg.de/rtems5-i368.tar.bz2, prefix like the one you have chosen: /home/travis/.rtems